### ISSUE TITLE
Support FROM clause for UPDATE and DELETE statements

### ISSUE THREAD

**Issue by slavaavr** [go-jet/jet#100]
Hi! Is any option to create such query as:
```
update table1 set a=table2.b
from table2
where table1.id = table2.fk_id
```
for postgres db?
What do you think about adding this feature? If you don't mind, I may help with the implementation.

**Comment by go-jet** [comment]
Hi, @slavaavr . `from` clause for `update` and `delete` statements are not currently supported. The workaround would be to use [IN operator](https://github.com/go-jet/jet/issues/29) or [raw statement](https://github.com/go-jet/jet/wiki/Statements#raw-statements).
Off course, feel free to contribute.

**Comment by go-jet** [comment]
This feature is now implemented in the `develop` branch and will be released later with release version 2.7.0.
